pkgpanda/build/test_util.py

```diff
@@ -0,0 +1,86 @@
+import pkgpanda.build
+
+
+def test_hash_files_in_folder(tmpdir):
+    hash_files_in_folder = pkgpanda.build.hash_files_in_folder
+
+    with tmpdir.as_cwd():
+        # Empty folder
```

Comment:
from my perspective, if the first test fails, _all_ the other ones are guaranteed to fail, and seeing their independent failure state will not provide useful help when debugging.

There would be some utility in making it easier to hand-run the different tests one by one for small iteration, but practically this is doing a "building up" / exercising more parts from case to case, hence why I made it one test.

Independent things get independent tests. 100 tests which will all fail because of the same single bug from my perspective produce a lot more noise than 1 test which builds up.